queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE %mon;
+SELECT * FROM animals WHERE date_of_birth BETWEEN CAST('2016-01-01' AS DATE) AND CAST('2019-12-30' AS DATE);
+SELECT * FROM animals WHERE neutered = false AND escape_attempts < 3;
```

Comment:
- [ ] Kindly make sure that neutered = true as the query needs to display all animals that are neutered.
- [ ] Make sure to add the updated screenshot for this query.